src/test/rustdoc/issue-66159.rs

```diff
@@ -1,10 +0,0 @@
-// aux-crate:priv:issue_66159_1=issue-66159-1.rs
-// compile-flags:-Z unstable-options
-
-// The issue was an ICE which meant that we never actually generated the docs
-// so if we have generated the docs, we're okay.
-// Since we don't generate the docs for the auxiliary files, we can't actually
-// verify that the struct is linked correctly.
-
-// @has issue_66159/index.html
-//! [issue_66159_1::Something]
```

Comment:
Perhaps it makes sense to keep this test even if the rest of the PR is reverted?

Reply:
Good catch, thanks. I'll add it back.